tests/test_cli_golden_contract.py

```
@@ -0,0 +1,471 @@
+    if "<agent>" in raw or raw.startswith("--") or raw.startswith("CMUX_"):
+        return []
+    raw = raw.removeprefix("cmux ").strip()
+    if not raw or raw.startswith("<"):
+        return []
+
+    tokens = raw.split()
+    literal_tokens: list[list[str]] = []
+    for token in tokens:
+        if token.startswith("<") or token.startswith("[") or token.startswith("("):
+            break
+        if token == "...":
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "token"

(S105)